app/scripts/modules/core/src/pipeline/config/stages/bakeManifest/helm/BakeHelmConfigForm.tsx

```diff
@@ -137,6 +138,16 @@ export class BakeHelmConfigForm extends React.Component<IFormikStageConfigInject
           pipeline={this.props.pipeline}
           stage={stage}
         />
+        {this.getInputArtifact(stage, 0).artifact.type == TYPE && (
```

Comment:
Very much open to suggestions of better ways to make this UI element visible only when the selected artifact type is git/repo